proxy/logging/unit-tests/test_LogUtils.cc

```diff
@@ -21,20 +21,70 @@
   limitations under the License.
  */
 
+#include <cstring>
+#include <cstdlib>
+#include <iostream>
+#include <cstdio>
+
 #include <tscore/ink_assert.h>
 #include <tscore/ink_align.h>
 
 #include <LogUtils.h>
 
-#include "test_LogUtils.h"
```

Comment:
Could you delete `test_LogUtils.h` file in this commit?

Reply:
Oops yes I meant to do that.

Reply:
No it turns out the include file is necessary.  It is a reduced version of MIME.h, because using the full MIME.h causes undefined symbols in the link.